src/transformers/utils/logging.py

```diff
@@ -163,3 +164,40 @@ def enable_propagation() -> None:
 
     _configure_library_root_logger()
     _get_library_root_logger().propagate = True
+
+
+log_levels = {
+    "debug": logging.DEBUG,
+    "info": logging.INFO,
+    "warning": logging.WARNING,
+    "error": logging.ERROR,
+    "critical": logging.CRITICAL,
+}
+
+
+def logging_levels_as_strings():
+    return log_levels.keys()
+
+
+def logging_level_str_to_code(level_str):
+    if level_str in log_levels:
+        return log_levels[level_str]
+    else:
+        raise ValueError(f"unknown level {level_str}, has to be one of: { log_levels.keys() }")
+
+
+def set_global_logging_level(level=logging.ERROR, prefices=[""]):
```

Comment:
What is the difference between this method and the previous `set_verbosity`?

I think we should select just one way to set the verbosity of the library.

Reply:
I will do some testing and report back.

Reply:
I resolved your suggestion, by leaving the `logging` API alone and instead adding all the custom functionality to `testing_utils` - this way this is a handy function for us when debugging things, but it really shouldn't be in the normal API.